.github/CODEOWNERS

```diff
@@ -428,6 +428,9 @@
 
 
 # Infrastructure.
+/beam_jobs/__init__.py @brianrodri
+/beam_jobs/base_model_validator.py @brianrodri
+/beam_jobs/base_model_validator_test.py @brianrodri
```

Comment:
Why use 'beam_jobs' and not just 'jobs'?

Reply:
Yup, we've changed to just `jobs`.